### ISSUE TITLE
Preview request

### ISSUE THREAD

**Issue by CreationCassel** [Zen4All-nl/Zen-Cart-Collect-Info-Through-Tabs-In-New-Style#1]
Would it be possible to have a preview in these tabs after all is "entered" and saved? The specific reasons are these:
- sometimes, accented characters might not show properly if not entered properly
- if graphics or videos are embedded in the description, it would give an idea what it would look like

This is just for convenience, before going back to the front end and see what it looks like and have to go back and edit whatever needs tweaking.

Thanks again for working on this. The backend has always been annoying to use, especially for multilanguage sites.

**Comment by Zen4All** [comment]
For now the default Zen Cart preview before saving  is used. 
The information presented there is pretty limited.
I am planing on adding all the entered data to show on the preview page.

> Thanks again for working on this. The backend has always been annoying to use, especially for multilanguage sites.

Yes I know. The idea started, when I had to build a Muli-store with 10 shops, and 3 languages. The page became like 3 miles long 😆

**Comment by CreationCassel** [comment]
> Yes I know. The idea started, when I had to build a Muli-store with 10 shops, and 3 languages. The page became like 3 miles long :laughing:

Yup. I have had that bilingual store for 8 years now, and adding multiple images was also a PITA.
